src/Transports.AspNetCore/GraphQLHttpMiddlewareOptions.cs

```diff
@@ -0,0 +1,107 @@
+#nullable enable
+
+using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+
+namespace GraphQL.Server.Transports.AspNetCore;
+
+/// <summary>
+/// Configuration options for <see cref="GraphQLHttpMiddleware"/>.
+/// </summary>
+public class GraphQLHttpMiddlewareOptions
+{
+    /// <summary>
+    /// Enables handling of GET requests.
+    /// </summary>
+    public bool HandleGet { get; set; } = true;
+
+    /// <summary>
+    /// Enables handling of POST requests, including form submissions, JSON-formatted requests and raw query requests.
```

Comment:
What is raw query?

Reply:
`application/graphql`, which is just the query string itself without variables or extensions or any escaping.

Reply:
ah, then I suggest to change this a bit

```suggestion
    /// Enables handling of POST requests, including form submissions (application/x-www-form-urlencoded), JSON-formatted requests (application/json) and raw (application/graphql) query requests.
```

Reply:
Form submissions also accept multipart or anything else understood by ASP.NET Core

Reply:
And shortly we will allow `application/graphql+json` as well.

Reply:
Did you mean to keep comment as is?

Reply:
bump

Reply:
I wouldn't change it.  There are at least two types allowed for form submissions, and shortly two for JSON formatted requests, plus the raw type.  I don't see that we really need to list every one of them.

Reply:
Or perhaps we add the inclusive list as a bulleted list appended to the xml comment.

Reply:
Updated with list format